app/javascript/readingList/readingList.jsx

```diff
@@ -117,6 +117,7 @@ export class ReadingList extends Component {
     this.setState({
       archiving: true,
       items: newItems,
+      itemsTotal: this.state.itemsTotal - 1,
```

Comment:
Nitpick: could you extract `itemsTotal` from `state` on line 109?

Reply:
cool. I'll apply the change.